### PR TITLE
Leaf/g1.4 init occupations and bridge table

### PR DESCRIPTION
**Pull request by shotoh**
#124 
#128

### CODE REVIEW

**Review by misslame — APPROVED** [review]
Woot LGTM

File: mesh/settings.py [hunk 1]
```diff
@@ -96,8 +96,8 @@
     'default': {
         'ENGINE': 'django.db.backends.mysql', 
         'NAME': 'mesh',
-        'USER': os.getenv("USER"),
-        'PASSWORD': os.getenv("PASS"), 
+        'USER': os.environ.get('DB_USER'),
+        'PASSWORD': os.environ.get('DB_PASS'),
```
Comment (peppacaiou) [inline mesh/settings.py:100]: if this gets merged, can you tell others to change their environment variable name

File: mesh/occupations/models.py [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+
+
 # Create your models here.
 class Occupation(models.Model):
-    #TODO
-    pass
+    occupationID = models.AutoField(primary_key=True)
+    occupationName = models.CharField(max_length=255)
+    occupationTag = models.CharField(max_length=255)
+    occupationDescriptor = models.TextField()
```
Comment (peppacaiou) [inline mesh/occupations/models.py:11]: should this have a maxlength?

**Review by peppacaiou — APPROVED** [review]
looks good, if my messages, don;t apply, this looks good, otherwise, a change would be great.

**Pull request merged**